minesweeper.py

```diff
@@ -160,9 +168,9 @@ def postcmd(self, stop, line):
         if stop:
             return True
 
-        print()
         if self.board:
-            print(self.board)
+            with self.term.location(0, 0):
+                print(self.board)
```

Comment:
This isn't robust against printing with an x-coordinate. Maybe extract a method or function to print at particular coordinates while aligning correctly on newline?